generated/xamarin-flexible-sync/realm-todo-dotnet/TaskPage.xaml.cs

```diff
@@ -83,19 +83,6 @@ private async void New_Button_Clicked(object sender, EventArgs e)
             });
         }
 
-        private void chkCompleted_PropertyChanged(object sender, PropertyChangedEventArgs e)
```

Comment:
I suppose this has been removed because it's done automatically through the bindings, am I correct?